package.json

```diff
@@ -12,42 +14,47 @@
     "postinstall-b:b": "git apply patches/patch-package+6.4.7.patch",
     "postinstall:c": "patch-package",
     "make-patch": "patch-package terraform --exclude='go.mod'",
-    "compile": "bash scripts/compile.sh",
+    "compile": "npm-run-all compile:*",
+    "compile:a": "bash scripts/compile.sh",
+    "compile:b": "webpack",
     "serve": "webpack-dev-server",
     "test": "exit 0"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/3p3r/tfweb-tfcli-wasml.git"
+    "url": "git+https://github.com/cdk-web/terraform.git"
   },
   "terraform": {
     "repo": "https://github.com/hashicorp/terraform",
     "branch": "v1.1.7"
   },
-  "keywords": [],
+  "keywords": [
+    "terraform",
+    "wasm",
+    "iac",
+    "self-service"
+  ],
   "author": "",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/3p3r/tfweb-tfcli-wasml/issues"
+    "url": "https://github.com/cdk-web/terraform/issues"
   },
-  "homepage": "https://github.com/3p3r/tfweb-tfcli-wasml#readme",
-  "dependencies": {
+  "homepage": "https://github.com/cdk-web/terraform#readme",
+  "devDependencies": {
     "@wasmer/wasmfs": "^0.12.0",
     "assert": "^2.0.0",
     "buffer": "^6.0.3",
     "crypto-browserify": "^3.12.0",
+    "http-serve": "^1.0.1",
```

Comment:
I added webpack dev server. Do we need both?

Reply:
this is still needed for jest-puppeteer added on later so it can serve the production minified bundle for puppeteer.